CacheManager.php

```diff
@@ -67,6 +67,8 @@ public function setGenerateUrlType($generateUrlType)
      *                           response
      *
      * @return $this
+     *
+     * @deprecated Add tags with TagHandler::addTags and then use TagHandler::tagResponse
```

Comment:
i don't go to the TagHandler here. so people who mix up new and old things might get into troubles. but otherwise we complicate things quite a bit.